Dockerfile

```diff
@@ -37,29 +38,30 @@ RUN apt-get update && \
                                                python3 \
                                                python3-pip \
                                                unzip \
-                                               yarn
-# No cleanup due to /etc/apt/apt.conf.d/docker-clean in ubuntu:20.04
+                                               yarn \
+ && apt-get clean \
+ && rm -rf /var/cache/apt/archives/*
 
 # binary downloads
 ARG TFSEC_VERSION="v0.24.1"
 ARG PACKER_VERSION="v1.6.1"
-RUN curl -L https://github.com/liamg/tfsec/releases/download/${TFSEC_VERSION}/tfsec-linux-amd64 -o /usr/local/bin/tfsec && \
-    chmod 0755 /usr/local/bin/tfsec && \
-    curl -L https://releases.hashicorp.com/packer/${PACKER_VERSION#v}/packer_${PACKER_VERSION#v}_linux_amd64.zip -o /usr/local/bin/packer.zip && \
-    unzip /usr/local/bin/packer.zip -d /usr/local/bin/ && \
-    chmod 0755 /usr/local/bin/packer
+RUN curl -L https://github.com/liamg/tfsec/releases/download/${TFSEC_VERSION}/tfsec-linux-amd64 -o /usr/local/bin/tfsec \
+ && chmod 0755 /usr/local/bin/tfsec \
+ && curl -L https://releases.hashicorp.com/packer/${PACKER_VERSION#v}/packer_${PACKER_VERSION#v}_linux_amd64.zip -o /usr/local/bin/packer.zip \
+ && unzip /usr/local/bin/packer.zip -d /usr/local/bin/ \
+ && chmod 0755 /usr/local/bin/packer
 
 # git installs
 ARG TERRAFORM_VERSION="0.12.29"
 ARG TFENV_VERSION="v2.0.0"
 ENV PATH="/root/.tfenv/bin:${PATH}"
-RUN git clone https://github.com/tfutils/tfenv.git ~/.tfenv && \
-    echo 'PATH=/root/.tfenv/bin:${PATH}' >> ~/.bashrc && \
-    . ~/.bashrc && \
-    cd ~/.tfenv && \
-    git checkout ${TFENV_VERSION} && \
-    tfenv install ${TERRAFORM_VERSION} && \
-    tfenv use ${TERRAFORM_VERSION}
+RUN git clone https://github.com/tfutils/tfenv.git ~/.tfenv \
+ && echo 'PATH=/root/.tfenv/bin:${PATH}' >> ~/.bashrc \
+ && . ~/.bashrc \
+ && cd ~/.tfenv \
```

Comment:
Any reason to `cd` here instead of setting a `WORKDIR` before the `RUN`? [Hadolint DL3003](https://github.com/hadolint/hadolint/wiki/DL3003)

Reply:
Yes, the `cd` is intentionally transient.